scripts/models/Image.js

```diff
@@ -0,0 +1,24 @@
+import { Media } from './Media.js';
+
+export class Image extends Media {
+  constructor(data) {
+    super(data);
+    if (data.image !== undefined) {
+      this._image = data.image;
+    } else {
+      this._image = '';
+    }
+    if (data.title !== undefined) {
+      this._title = data.title;
+    } else {
+      this._title = '';
+    }
```

Comment:
This can be simplified:
```suggestion
   this._image = data.image ?? ''
   this._title = data.title ?? ''
```